charts/kafka-connect-elastic5-sink/values.yaml

```diff
@@ -97,6 +97,13 @@ topics: "__REQUIRED__"
 # kcql KCQL expression describing field selection and routes. type: STRING importance: HIGH
 kcql: "__REQUIRED__"
 
+# A dictionary allowing to specify SMT's, e.g.
```

Comment:
we should leave this as an empty map in the default values file, users can always supply their own overrides.

Reply:
Was supposed to add it as a comment, rather than the default.